app/src/test/java/com/github/alfabravo2013/readyforexams/domain/login/LocalDataSourceTest.kt

```diff
@@ -33,19 +35,17 @@ internal class LocalDataSourceTest : KoinTest {
     inner class SignUpTest {
```

Comment:
I would add a DisplayName here too for SignUp and then inside every method tested I would only use `@DisplayName("Given ... Then ....")`